elixir/lib/homework/utils/pagination.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Homework.Utils.Pagination do
+
+  import Ecto.Query
+
+  def skip_limit(query, args) do
+    case args do
+      %{ pagination: %{ skip: s, limit: l } } -> query |> order_by(:inserted_at) |> offset(^s) |> limit(^l)
```

Comment:
The `order_by` ensures that when we offset/limit, we're paginating over new values.  This isn't the best approach, as we should be ordering by an indexed field for efficiency.  

This also has some drawbacks as it's not tamper-proof.  As our data is being mutated, our pagination may get messed up, and the way around that is to use a Keyset Pagination on an ordered index so that additions or updates don't throw off our pagination.